saleor/order/models.py

```diff
@@ -342,10 +343,7 @@ def can_mark_as_paid(self):
 
     @property
     def total_authorized(self):
-        payment = self.get_last_payment()
```

Comment:
We should also look at other usages of `get_last_payment` from Order model and replace it with loading payments with dataloader and using the utility function that this PR adds. I'm not saying we have to do it in this PR but we should keep it on the radar.

Reply:
Btw is the `total_authorized` property needed anymore after migrating to dataloader in the Order type?

Reply:
@maarcingebala About property - It is used but on subsequent PRs i'll probably be able to get rid of it, i also have it in mind, it's just used in some other properties which will also be changed to use DLs.